proposal/q40603/proposal.md

```diff
@@ -0,0 +1,97 @@
+Proposal for Pairs Trading 
```

Comment:
The template file uses [reStructuredText](https://docutils.sourceforge.io/rst.html), not markdown.  Is the filename a typo?

If you choose markdown, please reformat the file.